ext/gctrack/gctrack.c

```diff
@@ -40,6 +42,26 @@ add_gc_cycle(uint64_t duration)
   }
 }
 
+static void
+add_allocations()
```

Comment:
I think you can get that in `GC.stat(:total_allocated_objects)`

Reply:
I can't believe I overlooked that method, thanks Alex, that has more information that'd be super handy, great!